async port scaner.py

```diff
@@ -0,0 +1,31 @@
+import socket
```

Comment:
Rename the module to a pythonic name. i.e. `async_port_scanner.py`. We don't use spaces for module naming